File: W3/Count Number of Pairs With Absolute Difference K.dart
class Solution {
  int countKDifference(List<int> nums, int k) {
    int c=0;
    for(int i=0;i<nums.length;i++){
      for(int j=i+1;j<nums.length;j++){
        if(nums[i]-nums[j] == k || nums[i]-nums[j] == -k)
          c++;
      }
    }
    return c;
  }
}

File: W3/Count Number of Pairs With Absolute Difference K.cpp
class Solution {
public:
    int countKDifference(vector<int>& nums, int k) {
        map<int,int>fr;
        int c=0;
        for(int i:nums)
          fr[i]++;
        for(int i:nums){
          if(fr.find(i-k) != fr.end()){
            c += fr[i-k];
          }
        }
        return c;
    }
};